code/edgeidentity/src/test/java/com/adobe/marketing/mobile/edge/identity/IdentityItemTests.java

```diff
@@ -61,9 +71,10 @@ public void testIdentityItem_ctor2_emptyId() {
 		IdentityItem item = new IdentityItem("");
 
 		// For backward compatibility, an IdentityItem can contain empty identifiers.
-		assertEquals("", item.getId());
-		assertEquals("ambiguous", item.getAuthenticatedState().getName());
-		assertFalse(item.isPrimary());
+		String expected =
+			"{\n" + "  \"id\": \"\",\n" + "  \"authenticatedState\": \"ambiguous\",\n" + "  \"primary\": false\n" + "}";
```

Comment:
formatting

Reply:
When the id value is empty, SpotlessApply make this json into one line.

Reply:
ohkay